endpoints/openrtb2/auction.go

```diff
@@ -1510,12 +1510,12 @@ func (deps *endpointDeps) validateImpExt(imp *openrtb_ext.ImpWrapper, aliases ma
 	errL := []error{}
 
 	for bidder, ext := range prebid.Bidder {
-		coreBidder := bidder
+		coreBidder, _ := openrtb_ext.NormalizeBidderName(bidder)
```

Comment:
should check if `coreBidder` is returned from `openrtb_ext.NormalizeBidderName`

Reply:
This is already being checked down in the code via `deps.bidderMap[coreBidder.String()]`. It might also be the case where coreBidder is not being returned from `openrtb_ext.NormalizeBidderName` but rather an alias which is being set in line `1515`.
For above reasons, I have chosen to skip the check.

Reply:
>It might also be the case where coreBidder is not being returned from openrtb_ext.NormalizeBidderName but rather an alias which is being set in line 1515.

As per my understanding, `openrtb_ext.NormalizeBidderName` internally look up on `CoreBidderNames()` which includes alias as well as non alias bidders. Then will there be any case where alias will not present in `openrtb_ext.NormalizeBidderName` but in `aliases[bidder];` on line 1515

Reply:
yes. Publishers can choose to define their own aliases in the request. For example - publisher may choose to define an alias in `request.ext.aliases` as `"test1":"appnexus"`. Here `test1` is an alias of appnexus but not present in CoreBidderNames(). Please take a look at the sample request attached in the description that might help in getting the clarity.
Let me know if this makes sense.